roachtest: tpccbench/nodes=9/cpu=4/multi-region failed

SHA: https://github.com/cockroachdb/cockroach/commits/5612fc5f44e34cf10f60e63ed5a53b6dfa867190

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=tpccbench/nodes=9/cpu=4/multi-region PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1554254&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191023-1554254/tpccbench/nodes=9/cpu=4/multi-region/run_1
	test_runner.go:704: test timed out (10h0m0s)

```

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1649932&tab=buildLog) on [master@2f1e342c386973c35246bef68c177fcd0b8b609b](https://github.com/cockroachdb/cockroach/commits/2f1e342c386973c35246bef68c177fcd0b8b609b):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191218-1649932/tpccbench/nodes=9/cpu=4/multi-region/run_1
	cluster.go:2166,tpcc.go:720,tpcc.go:561,test_runner.go:700: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:720
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:561
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:700
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1337
		  - monitor failure:
		  - unexpected node event: 3: dead
```

<details><summary>Repro</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1649932&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
```
make stressrace TESTS=tpccbench/nodes=9/cpu=4/multi-region PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1653424&tab=buildLog) on [master@277e28b2ea78929563e85cb1c9efc573f37408a4](https://github.com/cockroachdb/cockroach/commits/277e28b2ea78929563e85cb1c9efc573f37408a4):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191219-1653424/tpccbench/nodes=9/cpu=4/multi-region/run_1
	cluster.go:2166,tpcc.go:720,tpcc.go:561,test_runner.go:700: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:720
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:561
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:700
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1337
		  - monitor failure:
		  - unexpected node event: 2: dead
```

<details><summary>Repro</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1653424&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
```
make stressrace TESTS=tpccbench/nodes=9/cpu=4/multi-region PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676715&tab=buildLog) on [master@52261c39c65aecf960e3c8d54089a66d378af610](https://github.com/cockroachdb/cockroach/commits/52261c39c65aecf960e3c8d54089a66d378af610):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200107-1676715/tpccbench/nodes=9/cpu=4/multi-region/run_1
	cluster.go:2189,tpcc.go:720,tpcc.go:561,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:720
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:561
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - unexpected node event: 7: dead
```

<details><summary>Repro</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676715&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
```
make stressrace TESTS=tpccbench/nodes=9/cpu=4/multi-region PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693614&tab=buildLog) on [master@e2c2be6cbef07c7f2ee6341816f81f72a206c32a](https://github.com/cockroachdb/cockroach/commits/e2c2be6cbef07c7f2ee6341816f81f72a206c32a):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200116-1693614/tpccbench/nodes=9/cpu=4/multi-region/run_1
	cluster.go:2201,tpcc.go:720,tpcc.go:561,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:720
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:561
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - unexpected node event: 6: dead
```

<details><summary>Repro</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693614&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
```
make stressrace TESTS=tpccbench/nodes=9/cpu=4/multi-region PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1702345&tab=buildLog) on [master@a5b42be47cf47206d2343f40bbaf445f387f3890](https://github.com/cockroachdb/cockroach/commits/a5b42be47cf47206d2343f40bbaf445f387f3890):

```
	cluster.go:2201,tpcc.go:720,tpcc.go:561,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:720
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:561
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		        main.execCmdWithBuffer
		        	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:419
		        main.(*cluster).RunWithBuffer
		        	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1867
		        main.loadTPCCBench
		        	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:647
		        main.runTPCCBench.func1
		        	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:718
		        main.(*monitor).Go.func1
		        	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		        github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
		        	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
		        runtime.goexit
		        	/usr/local/go/src/runtime/asm_amd64.s:1357
		      - error with embedded safe details: %s
		        -- arg 1: <string>
		      - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1702345-1579636752-85-n12cpu4-geo:4 -- ./workload run tpcc --warehouses=5000 --workers=5000 --max-rate=2500 --wait=false --duration=1h40m0s --scatter --tolerate-errors {pgurl:1-3,5-7,9-11}:
		      - exit status 1
		    failed with output "Initializing 5000 connections...\nInitializing 5000 workers and preparing statements...\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n    1.0s        0            0.0            0.0      0.0      0.0      0.0      0.0 delivery\n    1.0s        0            0.0            1.0    738.2    738.2    738.2    738.2 newOrder\n    1.0s        0            0.1           11.0    771.8    939.5    973.1    973.1 orderStatus\n    1.0s        0            0.1           10.0    704.6    939.5    939.5    939.5 payment\n    1.0s        0            0.0            4.0    604.0    805.3    805.3    805.3 stockLevel\n    2.0s        0            1.0            0.5   1208.0   1208.0   1208.0   1208.0 delivery\n    2.0s        0            2.0            1.5    570.4    872.4    872.4    872.4 newOrder\n    2.0s        0           41.0           26.0   1275.1   1677.7   1946.2   1946.2 orderStatus\n    2.0s        0          109.9           60.0   1140.9   1744.8   1879.0   1946.2 payment\n    2.0s        0           18.0           11.0   1208.0   1610.6   1610.6   1610.6 stockLevel\n    3.0s        0            0.0            0.3      0.0      0.0      0.0      0.0 delivery\n    3.0s        0            1.0            1.3   1476.4   1476.4   1476.4   1476.4 newOrder\n    3.0s        0           55.0           35.7   1744.8   2415.9   2415.9   2415.9 orderStatus\n    3.0s        0           89.0           69.7   1879.0   2684.4   2818.6   2952.8 payment\n    3.0s        0           43.0           21.7   1946.2   2550.1   2952.8   2952.8 stockLevel\n    4.0s        0            0.0            0.2      0.0      0.0      0.0      0.0 delivery\n    4.0s        0            3.0            1.7   1677.7   2147.5   2147.5   2147.5 newOrder\n    4.0s        0           72.9           45.0   2281.7   2952.8   3087.0   3087.0 orderStatus\n    4.0s        0          178.8           97.0   2952.8   3355.4   3489.7   3758.1 payment\n    4.0s        0           81.9           36.7   2550.1   3623.9   3892.3   3892.3 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n    5.0s        0            2.0            0.6   3892.3   4160.7   4160.7   4160.7 delivery\n    5.0s        0            6.0            2.6   4160.7   4563.4   4563.4   4563.4 newOrder\n    5.0s        0           45.9           45.2   2818.6   3489.7   3758.1   3758.1 orderStatus\n    5.0s        0          335.4          144.8   3758.1   4563.4   4831.8   4831.8 payment\n    5.0s        0           75.9           44.6   3221.2   4160.7   4295.0   4831.8 stockLevel\n    6.0s        0            0.0            0.5      0.0      0.0      0.0      0.0 delivery\n    6.0s        0           25.0            6.3   4563.4   5368.7   5368.7   5368.7 newOrder\n    6.0s        0           11.0           39.5   1744.8   2818.6   3355.4   3355.4 orderStatus\n    6.0s        0          279.9          167.3   4295.0   5100.3   5368.7   5368.7 payment\n    6.0s        0           24.0           41.2   3489.7   4563.4   4831.8   4831.8 stockLevel\n    7.0s        0            2.0            0.7   5905.6   6442.5   6442.5   6442.5 delivery\n    7.0s        0           77.0           16.4   6174.0   6710.9   6710.9   6979.3 newOrder\n    7.0s        0           12.0           35.6   1879.0   2952.8   3087.0   3087.0 orderStatus\n    7.0s        0          335.9          191.4   6442.5   6979.3   6979.3   6979.3 payment\n    7.0s        0            7.0           36.3   2281.7   3087.0   3087.0   3087.0 stockLevel\n    8.0s        0            4.0            1.1   7784.6   7784.6   7784.6   7784.6 delivery\n    8.0s        0           62.0           22.1   6979.3   7516.2   7516.2   7784.6 newOrder\n    8.0s        0           13.0           32.7   2281.7   3623.9   4563.4   4563.4 orderStatus\n    8.0s        0          286.1          203.2   6710.9   7247.8   7516.2   7784.6 payment\n    8.0s        0            7.0           32.6   2818.6   5100.3   5100.3   5100.3 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n    9.0s        0            6.0            1.7   8321.5   9126.8   9126.8   9126.8 delivery\n    9.0s        0           36.1           23.7   7784.6   8589.9   9126.8   9126.8 newOrder\n    9.0s        0           11.0           30.3   1946.2   3087.0   3758.1   3758.1 orderStatus\n    9.0s        0          126.2          194.7   7784.6   8321.5   8321.5   8321.5 payment\n    9.0s        0           12.0           30.3   2013.3   4026.5   5905.6   5905.6 stockLevel\n   10.0s        0            9.0            2.4   9126.8   9663.7   9663.7   9663.7 delivery\n   10.0s        0           96.0           30.9   8321.5   9126.8   9126.8   9126.8 newOrder\n   10.0s        0           15.0           28.8   2550.1   4160.7   5368.7   5368.7 orderStatus\n   10.0s        0          126.0          187.8   9126.8   9126.8   9126.8   9663.7 payment\n   10.0s        0           20.0           29.3   3087.0   5100.3   5368.7   5368.7 stockLevel\n   11.0s        0           13.0            3.4  10200.5  10737.4  10737.4  10737.4 delivery\n   11.0s        0           64.0           33.9   9663.7  10737.4  10737.4  11274.3 newOrder\n   11.0s        0           13.0           27.4   2818.6   3758.1   5905.6   5905.6 orderStatus\n   11.0s        0          182.9          187.4   9126.8   9663.7   9663.7  10200.5 payment\n   11.0s        0            7.0           27.3   2952.8   3892.3   3892.3   3892.3 stockLevel\n   12.0s        0           17.0            4.5  11274.3  11811.2  11811.2  11811.2 delivery\n   12.0s        0          212.0           48.7  10737.4  11274.3  11274.3  11811.2 newOrder\n   12.0s        0           19.0           26.7   2013.3   4160.7   4563.4   4563.4 orderStatus\n   12.0s        0          138.0          183.2   7247.8  10737.4  10737.4  10737.4 payment\n   12.0s        0            8.0           25.7   1610.6   5637.1   5637.1   5637.1 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   13.0s        0           39.9            7.2  11811.2  12884.9  13421.8  13421.8 delivery\n   13.0s        0           23.0           46.8  11274.3  12348.0  12884.9  12884.9 newOrder\n   13.0s        0            8.0           25.2    671.1   2550.1   2550.1   2550.1 orderStatus\n   13.0s        0          324.5          194.2   8589.9  12348.0  12348.0  12884.9 payment\n   13.0s        0            4.0           24.0   1140.9   6174.0   6174.0   6174.0 stockLevel\n   14.0s        0            4.0            7.0  12348.0  13421.8  13421.8  13421.8 delivery\n   14.0s        0           25.1           45.2  12348.0  13421.8  13958.6  13958.6 newOrder\n   14.0s        0            4.0           23.7   1140.9   5637.1   5637.1   5637.1 orderStatus\n   14.0s        0           50.1          183.9  12348.0  13421.8  13421.8  13421.8 payment\n   14.0s        0            3.0           22.5   5100.3   6710.9   6710.9   6710.9 stockLevel\n   15.0s        0            5.0            6.9   8589.9  13421.8  13421.8  13421.8 delivery\n   15.0s        0           15.0           43.2  11274.3  13421.8  13958.6  13958.6 newOrder\n   15.0s        0            6.0           22.5   4026.5   7784.6   7784.6   7784.6 orderStatus\n   15.0s        0          127.9          180.1  12348.0  13421.8  13421.8  13421.8 payment\n   15.0s        0            7.0           21.5   3758.1   6979.3   6979.3   6979.3 stockLevel\n   16.0s        0            8.0            6.9  10737.4  15569.3  15569.3  15569.3 delivery\n   16.0s        0           22.0           41.9  11811.2  15569.3  15569.3  15569.3 newOrder\n   16.0s        0            5.0           21.4   1040.2   5905.6   5905.6   5905.6 orderStatus\n   16.0s        0           43.0          171.6  11274.3  14495.5  14495.5  14495.5 payment\n   16.0s        0            6.0           20.5   1543.5   9126.8   9126.8   9126.8 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   17.0s        0            6.0            6.9  15032.4  17179.9  17179.9  17179.9 delivery\n   17.0s        0           39.0           41.7  15569.3  16106.1  16643.0  16643.0 newOrder\n   17.0s        0            0.0           20.2      0.0      0.0      0.0      0.0 orderStatus\n   17.0s        0           17.0          162.5   4563.4  10200.5  12348.0  12348.0 payment\n   17.0s        0            4.0           19.5   5368.7   9663.7   9663.7   9663.7 stockLevel\n   18.0s        0            7.0            6.9  12348.0  16643.0  16643.0  16643.0 delivery\n   18.0s        0           36.0           41.4  16643.0  18253.6  18253.6  18253.6 newOrder\n   18.0s        0            0.0           19.1      0.0      0.0      0.0      0.0 orderStatus\n   18.0s        0           35.0          155.4  11274.3  16643.0  16643.0  16643.0 payment\n   18.0s        0            4.0           18.7   4026.5   7247.8   7247.8   7247.8 stockLevel\n   19.0s        0           14.0            7.3  13958.6  18253.6  18253.6  18253.6 delivery\n   19.0s        0          225.6           51.1  18253.6  19327.4  19327.4  19327.4 newOrder\n   19.0s        0            8.0           18.5   5905.6   6710.9   6710.9   6710.9 orderStatus\n   19.0s        0           58.9          150.3  13421.8  16643.0  17179.9  17179.9 payment\n   19.0s        0           10.0           18.2   5100.3  10200.5  10200.5  10200.5 stockLevel\n   20.0s        0           29.0            8.3  18253.6  19327.4  19327.4  19327.4 delivery\n   20.0s        0           98.1           53.4  19327.4  20401.1  20401.1  20401.1 newOrder\n   20.0s        0            7.0           17.9   5637.1   7784.6   7784.6   7784.6 orderStatus\n   20.0s        0          105.1          148.0  10200.5  19327.4  19327.4  19327.4 payment\n   20.0s        0            2.0           17.4   5637.1   7516.2   7516.2   7516.2 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   21.0s        0           14.0            8.6  19327.4  20401.1  20401.1  20401.1 delivery\n   21.0s        0           53.0           53.4  19327.4  20401.1  21474.8  21474.8 newOrder\n   21.0s        0            5.0           17.3   1409.3  10737.4  10737.4  10737.4 orderStatus\n   21.0s        0           84.0          145.0   9126.8  18253.6  20401.1  20401.1 payment\n   21.0s        0           14.0           17.2   3892.3   8589.9  12884.9  12884.9 stockLevel\n   22.0s        0           15.9            9.0  20401.1  21474.8  21474.8  21474.8 delivery\n   22.0s        0           69.8           54.2  20401.1  21474.8  22548.6  22548.6 newOrder\n   22.0s        0           12.0           17.0   1677.7   9126.8  11811.2  11811.2 orderStatus\n   22.0s        0          329.9          153.5  12884.9  20401.1  20401.1  20401.1 payment\n   22.0s        0           22.9           17.5   2818.6  10737.4  16643.0  16643.0 stockLevel\n   23.0s        0           16.0            9.3  17179.9  22548.6  22548.6  22548.6 delivery\n   23.0s        0          187.3           60.0  21474.8  22548.6  23622.3  23622.3 newOrder\n   23.0s        0           22.0           17.3   3221.2  12348.0  13421.8  13421.8 orderStatus\n   23.0s        0          169.3          154.1  10737.4  16643.0  18253.6  18253.6 payment\n   23.0s        0           16.0           17.4   2952.8   4563.4  12348.0  12348.0 stockLevel\n   24.0s        0           11.0            9.3  19327.4  22548.6  23622.3  23622.3 delivery\n   24.0s        0          188.0           65.3  22548.6  23622.3  23622.3  23622.3 newOrder\n   24.0s        0           18.0           17.3   1342.2   3221.2   4295.0   4295.0 orderStatus\n   24.0s        0          160.0          154.4   5637.1  16106.1  18253.6  18253.6 payment\n   24.0s        0           18.0           17.5   2281.7  13421.8  15569.3  15569.3 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   25.0s        0           13.0            9.5  15569.3  23622.3  24696.1  24696.1 delivery\n   25.0s        0          150.0           68.7  19327.4  24696.1  25769.8  25769.8 newOrder\n   25.0s        0           13.0           17.1   1476.4   3623.9  14495.5  14495.5 orderStatus\n   25.0s        0          140.0          153.8   4563.4  15032.4  18253.6  19327.4 payment\n   25.0s        0           13.0           17.3   1946.2   2818.6  13421.8  13421.8 stockLevel\n   26.0s        0           13.0            9.6  18253.6  21474.8  22548.6  22548.6 delivery\n   26.0s        0          267.7           76.3  18253.6  24696.1  25769.8  26843.5 newOrder\n   26.0s        0           26.0           17.5   1677.7   6710.9  14495.5  14495.5 orderStatus\n   26.0s        0          169.8          154.4   3489.7  11811.2  19327.4  19327.4 payment\n   26.0s        0           19.0           17.3   2281.7  19327.4  19327.4  19327.4 stockLevel\n   27.0s        0           14.0            9.8  16106.1  24696.1  25769.8  25769.8 delivery\n   27.0s        0          218.1           81.6  14495.5  25769.8  26843.5  26843.5 newOrder\n   27.0s        0           14.0           17.3   1140.9   1744.8   2147.5   2147.5 orderStatus\n   27.0s        0          119.0          153.1   3087.0   7516.2  13958.6  16643.0 payment\n   27.0s        0           11.0           17.1   1946.2   4563.4   5368.7   5368.7 stockLevel\n   28.0s        0           21.0           10.2  17179.9  23622.3  27917.3  27917.3 delivery\n   28.0s        0          281.8           88.7   9126.8  22548.6  26843.5  27917.3 newOrder\n   28.0s        0           21.0           17.5   1610.6   3892.3   6442.5   6442.5 orderStatus\n   28.0s        0          175.9          153.9   5368.7  18253.6  21474.8  23622.3 payment\n   28.0s        0           23.0           17.3   2147.5  17179.9  18253.6  18253.6 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   29.0s        0            9.0           10.1  16643.0  27917.3  27917.3  27917.3 delivery\n   29.0s        0          186.9           92.1   7784.6  27917.3  28991.0  28991.0 newOrder\n   29.0s        0           16.0           17.4   1342.2   3221.2   5368.7   5368.7 orderStatus\n   29.0s        0          373.7          161.5   5368.7  22548.6  23622.3  23622.3 payment\n   29.0s        0           20.0           17.4   1946.2   4831.8   5368.7   5368.7 stockLevel\n   30.0s        0           17.0           10.4  10200.5  28991.0  28991.0  28991.0 delivery\n   30.0s        0          309.9           99.4   8053.1  30064.8  30064.8  30064.8 newOrder\n   30.0s        0           34.0           18.0   1610.6  11274.3  14495.5  14495.5 orderStatus\n   30.0s        0           70.0          158.5   4160.7  22548.6  23622.3  26843.5 payment\n   30.0s        0           14.0           17.3   2415.9   6174.0  22548.6  22548.6 stockLevel\n   31.0s        0           16.0           10.5  30064.8  31138.5  31138.5  31138.5 delivery\n   31.0s        0          296.0          105.7  28991.0  31138.5  31138.5  31138.5 newOrder\n   31.0s        0           14.0           17.8   2281.7  10737.4  19327.4  19327.4 orderStatus\n   31.0s        0          114.0          157.0   4831.8  24696.1  25769.8  30064.8 payment\n   31.0s        0           14.0           17.2   6174.0  16643.0  20401.1  20401.1 stockLevel\n   32.0s        0           17.0           10.7  30064.8  32212.3  32212.3  32212.3 delivery\n   32.0s        0          130.3          106.5  30064.8  31138.5  32212.3  32212.3 newOrder\n   32.0s        0           24.1           18.0   1744.8  20401.1  20401.1  20401.1 orderStatus\n   32.0s        0           43.1          153.5   4563.4  20401.1  25769.8  25769.8 payment\n   32.0s        0           20.0           17.3   9126.8  20401.1  23622.3  23622.3 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   33.0s        0           15.0           10.9  31138.5  33286.0  33286.0  33286.0 delivery\n   33.0s        0          209.9          109.6  30064.8  32212.3  33286.0  33286.0 newOrder\n   33.0s        0           11.0           17.8   2818.6  11811.2  25769.8  25769.8 orderStatus\n   33.0s        0           69.0          150.9   5368.7  21474.8  23622.3  27917.3 payment\n   33.0s        0           34.0           17.8   3355.4  18253.6  21474.8  21474.8 stockLevel\n   34.0s        0           19.0           11.1  32212.3  33286.0  33286.0  33286.0 delivery\n   34.0s        0          200.9          112.3  32212.3  33286.0  34359.7  34359.7 newOrder\n   34.0s        0           17.0           17.8   1677.7  11811.2  12348.0  12348.0 orderStatus\n   34.0s        0           97.0          149.3   7784.6  28991.0  30064.8  30064.8 payment\n   34.0s        0           22.0           17.9   3087.0  15569.3  24696.1  24696.1 stockLevel\n   35.0s        0           18.0           11.3  12348.0  34359.7  36507.2  36507.2 delivery\n   35.0s        0          166.1          113.9  33286.0  36507.2  36507.2  36507.2 newOrder\n   35.0s        0           19.0           17.8   3087.0  12884.9  22548.6  22548.6 orderStatus\n   35.0s        0           87.1          147.5   7247.8  28991.0  31138.5  31138.5 payment\n   35.0s        0           16.0           17.9   3221.2   6174.0   6710.9   6710.9 stockLevel\n   36.0s        0           23.0           11.6  21474.8  36507.2  36507.2  36507.2 delivery\n   36.0s        0          157.0          115.1  28991.0  36507.2  36507.2  36507.2 newOrder\n   36.0s        0           16.0           17.8   1342.2   3758.1   4831.8   4831.8 orderStatus\n   36.0s        0           49.0          144.8   7784.6  28991.0  31138.5  31138.5 payment\n   36.0s        0           14.0           17.7   2415.9  23622.3  24696.1  24696.1 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   37.0s        0           19.0           11.8  25769.8  36507.2  36507.2  36507.2 delivery\n   37.0s        0          254.5          118.8  24696.1  38654.7  38654.7  38654.7 newOrder\n   37.0s        0           18.0           17.8   1610.6   6979.3   6979.3   6979.3 orderStatus\n   37.0s        0          145.7          144.8   9663.7  30064.8  32212.3  32212.3 payment\n   37.0s        0            7.0           17.5   2281.7   8321.5   8321.5   8321.5 stockLevel\n   38.0s        0           30.9           12.3  15569.3  36507.2  38654.7  38654.7 delivery\n   38.0s        0          365.9          125.4  25769.8  36507.2  38654.7  38654.7 newOrder\n   38.0s        0           17.9           17.8   1744.8   3892.3   5368.7   5368.7 orderStatus\n   38.0s        0          402.8          151.7  11811.2  31138.5  33286.0  34359.7 payment\n   38.0s        0           15.0           17.4   2952.8   6442.5   7516.2   7516.2 stockLevel\n   39.0s        0           20.0           12.5  13958.6  38654.7  38654.7  38654.7 delivery\n   39.0s        0          475.0          134.3  24696.1  38654.7  38654.7  40802.2 newOrder\n   39.0s        0           35.0           18.2   1543.5   4295.0   9126.8   9126.8 orderStatus\n   39.0s        0          309.0          155.7   9126.8  27917.3  32212.3  33286.0 payment\n   39.0s        0           20.0           17.5   3355.4  10737.4  11274.3  11274.3 stockLevel\n   40.0s        0           21.1           12.7  15569.3  33286.0  38654.7  38654.7 delivery\n   40.0s        0          143.4          134.5  18253.6  33286.0  40802.2  40802.2 newOrder\n   40.0s        0           29.1           18.5   1744.8   3623.9   5100.3   5100.3 orderStatus\n   40.0s        0          184.6          156.4   6979.3  11811.2  26843.5  32212.3 payment\n   40.0s        0           25.1           17.6   2281.7   7516.2  10200.5  10200.5 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   41.0s        0           12.0           12.7  15569.3  25769.8  28991.0  28991.0 delivery\n   41.0s        0          331.7          139.4  17179.9  36507.2  40802.2  40802.2 newOrder\n   41.0s        0           17.0           18.5   1946.2   2550.1   3489.7   3489.7 orderStatus\n   41.0s        0          109.9          155.3   5905.6  11811.2  28991.0  30064.8 payment\n   41.0s        0           26.0           17.9   2415.9   3087.0   3221.2   3221.2 stockLevel\n   42.0s        0           20.0           12.9  16106.1  38654.7  38654.7  38654.7 delivery\n   42.0s        0           63.1          137.5  13958.6  34359.7  36507.2  42949.7 newOrder\n   42.0s        0           17.0           18.4   1543.5   3623.9   3623.9   3623.9 orderStatus\n   42.0s        0          269.4          158.0   6979.3  25769.8  34359.7  38654.7 payment\n   42.0s        0           34.0           18.2   2952.8   8589.9  10200.5  10200.5 stockLevel\n   43.0s        0           34.0           13.4  15569.3  32212.3  38654.7  38654.7 delivery\n   43.0s        0          294.8          141.2  14495.5  34359.7  38654.7  40802.2 newOrder\n   43.0s        0           17.0           18.4   1946.2   3489.7   5637.1   5637.1 orderStatus\n   43.0s        0          128.9          157.3   6174.0  12348.0  18253.6  18253.6 payment\n   43.0s        0           16.0           18.2   2818.6   6174.0  13958.6  13958.6 stockLevel\n   44.0s        0           24.0           13.6  17179.9  34359.7  36507.2  36507.2 delivery\n   44.0s        0           81.0          139.8  12884.9  33286.0  36507.2  38654.7 newOrder\n   44.0s        0            5.0           18.1   1073.7   3758.1   3758.1   3758.1 orderStatus\n   44.0s        0          324.0          161.1   9126.8  18253.6  30064.8  38654.7 payment\n   44.0s        0           16.0           18.1   3087.0   5100.3   5637.1   5637.1 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   45.0s        0           16.0           13.7  13958.6  26843.5  33286.0  33286.0 delivery\n   45.0s        0          113.0          139.2  14495.5  33286.0  40802.2  40802.2 newOrder\n   45.0s        0            3.0           17.8   1946.2   3892.3   3892.3   3892.3 orderStatus\n   45.0s        0          244.0          162.9   5905.6  16106.1  36507.2  38654.7 payment\n   45.0s        0           10.0           18.0   2550.1  23622.3  23622.3  23622.3 stockLevel\n   46.0s        0           12.0           13.7  14495.5  20401.1  36507.2  36507.2 delivery\n   46.0s        0          167.1          139.8  13421.8  33286.0  40802.2  40802.2 newOrder\n   46.0s        0           12.0           17.6    704.6   1409.3   2550.1   2550.1 orderStatus\n   46.0s        0          296.1          165.8   7784.6  17179.9  20401.1  40802.2 payment\n   46.0s        0            7.0           17.7   1275.1   3355.4   3355.4   3355.4 stockLevel\n   47.0s        0           22.0           13.8  10200.5  18253.6  19327.4  19327.4 delivery\n   47.0s        0           40.0          137.7   6442.5  16106.1  18253.6  18253.6 newOrder\n   47.0s        0            7.0           17.4    973.1   1543.5   1543.5   1543.5 orderStatus\n   47.0s        0          198.2          166.5   3623.9   9126.8  13958.6  15032.4 payment\n   47.0s        0           18.0           17.7   1744.8   5637.1  18253.6  18253.6 stockLevel\n   48.0s        0           20.0           14.0  11811.2  19327.4  21474.8  21474.8 delivery\n   48.0s        0          142.9          137.8   7784.6  12348.0  14495.5  15032.4 newOrder\n   48.0s        0            5.0           17.1   1040.2   5905.6   5905.6   5905.6 orderStatus\n   48.0s        0          122.9          165.6   2550.1   5637.1   8053.1  10200.5 payment\n   48.0s        0           11.0           17.6   6174.0  12348.0  12348.0  12348.0 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   49.0s        0           31.0           14.3   8053.1  12348.0  13958.6  13958.6 delivery\n   49.0s        0           42.0          135.9   5905.6  16106.1  18253.6  18253.6 newOrder\n   49.0s        0            7.0           16.9    939.5   6174.0   6174.0   6174.0 orderStatus\n   49.0s        0          160.2          165.5   3221.2  11811.2  20401.1  45097.2 payment\n   49.0s        0           10.0           17.4   1342.2   6174.0   6174.0   6174.0 stockLevel\n   50.0s        0           10.0           14.2   9663.7  42949.7  42949.7  42949.7 delivery\n   50.0s        0           94.9          135.1  13958.6  21474.8  42949.7  47244.6 newOrder\n   50.0s        0           27.0           17.1   4563.4   9126.8   9663.7   9663.7 orderStatus\n   50.0s        0          187.8          165.9  10737.4  20401.1  25769.8  45097.2 payment\n   50.0s        0           12.0           17.3   6174.0   8053.1  11274.3  11274.3 stockLevel\n   51.0s        0            6.0           14.1  13958.6  21474.8  21474.8  21474.8 delivery\n   51.0s        0          149.9          135.4  15032.4  22548.6  24696.1  42949.7 newOrder\n   51.0s        0           14.0           17.1   3489.7   8053.1   9663.7   9663.7 orderStatus\n   51.0s        0          100.9          164.7  10200.5  15569.3  19327.4  23622.3 payment\n   51.0s        0            6.0           17.1   3758.1  12884.9  12884.9  12884.9 stockLevel\n   52.0s        0           15.0           14.1  17179.9  25769.8  40802.2  40802.2 delivery\n   52.0s        0          114.1          134.9  16643.0  23622.3  47244.6  47244.6 newOrder\n   52.0s        0           14.0           17.0   5100.3  10200.5  10200.5  10200.5 orderStatus\n   52.0s        0          183.1          165.0   5637.1  15032.4  17179.9  42949.7 payment\n   52.0s        0           12.0           17.0   5637.1   9126.8  10200.5  10200.5 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   53.0s        0           17.0           14.1  16643.0  22548.6  24696.1  24696.1 delivery\n   53.0s        0          127.9          134.8  15032.4  23622.3  25769.8  25769.8 newOrder\n   53.0s        0           15.0           17.0   2818.6   6442.5   8053.1   8053.1 orderStatus\n   53.0s        0          195.8          165.6   7784.6  15569.3  16643.0  25769.8 payment\n   53.0s        0           17.0           17.0   3087.0  11274.3  14495.5  14495.5 stockLevel\n   54.0s        0           22.0           14.3  17179.9  25769.8  26843.5  26843.5 delivery\n   54.0s        0          291.0          137.7  15569.3  23622.3  25769.8  47244.6 newOrder\n   54.0s        0           42.0           17.4   4563.4  11811.2  12884.9  12884.9 orderStatus\n   54.0s        0          224.0          166.7   8589.9  18253.6  25769.8  25769.8 payment\n   54.0s        0            5.0           16.8   2080.4   4831.8   4831.8   4831.8 stockLevel\n   55.0s        0           25.0           14.5  17179.9  25769.8  27917.3  27917.3 delivery\n   55.0s        0          237.7          139.5  16643.0  22548.6  26843.5  53687.1 newOrder\n   55.0s        0           21.0           17.5   2550.1   9663.7  12884.9  12884.9 orderStatus\n   55.0s        0           90.9          165.3   5637.1  17179.9  23622.3  26843.5 payment\n   55.0s        0            8.0           16.6   1610.6  11811.2  11811.2  11811.2 stockLevel\n   56.0s        0           15.0           14.5  17179.9  26843.5  27917.3  27917.3 delivery\n   56.0s        0          126.1          139.3  17179.9  26843.5  45097.2  45097.2 newOrder\n   56.0s        0           35.0           17.8   1744.8  15032.4  18253.6  18253.6 orderStatus\n   56.0s        0          153.2          165.1   4831.8  22548.6  27917.3  27917.3 payment\n   56.0s        0           22.0           16.7   2080.4   3758.1   7247.8   7247.8 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   57.0s        0           12.0           14.4  19327.4  30064.8  42949.7  42949.7 delivery\n   57.0s        0          175.2          139.9  18253.6  27917.3  28991.0  28991.0 newOrder\n   57.0s        0           19.0           17.8   4831.8  15032.4  19327.4  19327.4 orderStatus\n   57.0s        0           93.1          163.8   4563.4  20401.1  28991.0  31138.5 payment\n   57.0s        0           18.0           16.7   3221.2  15569.3  21474.8  21474.8 stockLevel\n   58.0s        0           15.0           14.4  12348.0  22548.6  28991.0  28991.0 delivery\n   58.0s        0          130.0          139.7  12884.9  27917.3  28991.0  30064.8 newOrder\n   58.0s        0           15.0           17.8   2550.1   9126.8  15032.4  15032.4 orderStatus\n   58.0s        0           75.0          162.3   4026.5   7247.8  20401.1  21474.8 payment\n   58.0s        0           16.0           16.7   7784.6  15569.3  15569.3  15569.3 stockLevel\n   59.0s        0           14.0           14.4  15032.4  28991.0  32212.3  32212.3 delivery\n   59.0s        0           53.0          138.3   7247.8  23622.3  24696.1  28991.0 newOrder\n   59.0s        0           11.0           17.7   1946.2  15569.3  18253.6  18253.6 orderStatus\n   59.0s        0          101.0          161.3   5100.3  21474.8  30064.8  31138.5 payment\n   59.0s        0           21.0           16.8   3758.1  16106.1  16106.1  16106.1 stockLevel\n   60.0s        0           12.0           14.4  10737.4  20401.1  34359.7  34359.7 delivery\n   60.0s        0          146.0          138.4   9663.7  27917.3  31138.5  38654.7 newOrder\n   60.0s        0           15.0           17.6   1476.4  10200.5  16643.0  16643.0 orderStatus\n   60.0s        0          165.0          161.3   5368.7  10737.4  16106.1  22548.6 payment\n   60.0s        0           21.0           16.9   3489.7  16106.1  16643.0  16643.0 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   61.0s        0            5.0           14.2  12884.9  24696.1  24696.1  24696.1 delivery\n   61.0s        0          201.6          139.4   7784.6  25769.8  33286.0  55834.6 newOrder\n   61.0s        0           12.0           17.5   2684.4  11274.3  21474.8  21474.8 orderStatus\n   61.0s        0          166.6          161.4   6174.0  15032.4  20401.1  23622.3 payment\n   61.0s        0           18.0           16.9   4831.8  19327.4  24696.1  24696.1 stockLevel\n   62.0s        0           15.0           14.3  13421.8  23622.3  26843.5  26843.5 delivery\n   62.0s        0          110.1          139.0  11811.2  27917.3  33286.0  51539.6 newOrder\n   62.0s        0           19.0           17.6   1811.9  19327.4  23622.3  23622.3 orderStatus\n   62.0s        0          127.1          160.9   9126.8  22548.6  24696.1  24696.1 payment\n   62.0s        0            7.0           16.7   7516.2  22548.6  22548.6  22548.6 stockLevel\n   63.0s        0           16.0           14.3  15032.4  34359.7  38654.7  38654.7 delivery\n   63.0s        0          228.8          140.4  16106.1  25769.8  27917.3  28991.0 newOrder\n   63.0s        0           13.0           17.5   1610.6   3892.3  18253.6  18253.6 orderStatus\n   63.0s        0          225.8          161.9   9663.7  23622.3  27917.3  33286.0 payment\n   63.0s        0           21.0           16.8   3087.0  13421.8  22548.6  22548.6 stockLevel\n   64.0s        0           15.0           14.3  22548.6  51539.6  53687.1  53687.1 delivery\n   64.0s        0          297.5          142.9  15569.3  26843.5  28991.0  31138.5 newOrder\n   64.0s        0           21.0           17.5   1275.1   2684.4   4295.0   4295.0 orderStatus\n   64.0s        0          131.8          161.4   5100.3  13421.8  23622.3  25769.8 payment\n   64.0s        0           22.0           16.9   2952.8  21474.8  24696.1  24696.1 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   65.0s        0           13.0           14.3  11274.3  30064.8  45097.2  45097.2 delivery\n   65.0s        0          148.3          142.9  13958.6  27917.3  34359.7  36507.2 newOrder\n   65.0s        0           17.0           17.5   1543.5   3758.1   8053.1   8053.1 orderStatus\n   65.0s        0          155.3          161.3   7247.8  20401.1  24696.1  27917.3 payment\n   65.0s        0           17.0           16.9   2281.7  20401.1  21474.8  21474.8 stockLevel\n   66.0s        0           18.0           14.3  17179.9  34359.7  47244.6  47244.6 delivery\n   66.0s        0          353.2          146.1  22548.6  31138.5  38654.7  57982.1 newOrder\n   66.0s        0           19.0           17.6   1744.8   2818.6   6979.3   6979.3 orderStatus\n   66.0s        0           59.9          159.8   5905.6  20401.1  25769.8  25769.8 payment\n   66.0s        0           14.0           16.8   2281.7   5905.6  10737.4  10737.4 stockLevel\n   67.0s        0           16.0           14.4  14495.5  36507.2  45097.2  45097.2 delivery\n   67.0s        0          164.4          146.4  17179.9  30064.8  32212.3  32212.3 newOrder\n   67.0s        0           10.0           17.4   1811.9   3355.4   3355.4   3355.4 orderStatus\n   67.0s        0           54.1          158.2   5637.1  24696.1  30064.8  31138.5 payment\n   67.0s        0           24.1           16.9   2952.8   5100.3  26843.5  26843.5 stockLevel\n   68.0s        0           24.0           14.5  21474.8  62277.0  66572.0  66572.0 delivery\n   68.0s        0          234.8          147.7  16106.1  31138.5  36507.2  40802.2 newOrder\n   68.0s        0            2.0           17.2   2080.4   5368.7   5368.7   5368.7 orderStatus\n   68.0s        0          189.8          158.7   7516.2  30064.8  31138.5  34359.7 payment\n   68.0s        0           29.0           17.1   2415.9  17179.9  25769.8  25769.8 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   69.0s        0            7.0           14.4  36507.2  42949.7  42949.7  42949.7 delivery\n   69.0s        0          152.2          147.8  14495.5  31138.5  32212.3  36507.2 newOrder\n   69.0s        0           14.0           17.2   1476.4   3892.3   5100.3   5100.3 orderStatus\n   69.0s        0          100.1          157.8   6979.3  21474.8  26843.5  36507.2 payment\n   69.0s        0           22.0           17.2   2818.6  12884.9  23622.3  23622.3 stockLevel\n   70.0s        0           31.0           14.6  27917.3  42949.7  42949.7  42949.7 delivery\n   70.0s        0           14.0          145.9  13958.6  33286.0  40802.2  40802.2 newOrder\n   70.0s        0            2.0           17.0    402.7  27917.3  27917.3  27917.3 orderStatus\n   70.0s        0          116.1          157.2  23622.3  30064.8  32212.3  34359.7 payment\n   70.0s        0            7.0           17.0   3221.2   7247.8   7247.8   7247.8 stockLevel\n   71.0s        0           13.0           14.6  17179.9  38654.7  40802.2  40802.2 delivery\n   71.0s        0           54.0          144.6  15032.4  34359.7  38654.7  42949.7 newOrder\n   71.0s        0            6.0           16.8   1946.2   3489.7   3489.7   3489.7 orderStatus\n   71.0s        0          115.0          156.6  11274.3  33286.0  42949.7  64424.5 payment\n   71.0s        0            9.0           16.9   8321.5  18253.6  18253.6  18253.6 stockLevel\n   72.0s        0            8.0           14.5  11811.2  36507.2  36507.2  36507.2 delivery\n   72.0s        0           26.0          142.9  16106.1  34359.7  34359.7  34359.7 newOrder\n   72.0s        0            5.0           16.6   1275.1   4160.7   4160.7   4160.7 orderStatus\n   72.0s        0          143.0          156.4   8589.9  26843.5  64424.5  64424.5 payment\n   72.0s        0           12.0           16.9   4563.4  10737.4  13421.8  13421.8 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   73.0s        0           25.0           14.7  18253.6  36507.2  36507.2  36507.2 delivery\n   73.0s        0          118.9          142.6  18253.6  36507.2  42949.7  49392.1 newOrder\n   73.0s        0           11.0           16.6   5905.6   6979.3  13421.8  13421.8 orderStatus\n   73.0s        0          109.9          155.8   6979.3  32212.3  36507.2  36507.2 payment\n   73.0s        0           18.0           16.9   5637.1  11274.3  12348.0  12348.0 stockLevel\n   74.0s        0           14.0           14.6  13958.6  28991.0  36507.2  36507.2 delivery\n   74.0s        0           31.0          141.1  21474.8  36507.2  36507.2  36507.2 newOrder\n   74.0s        0            6.0           16.4   2147.5  10200.5  10200.5  10200.5 orderStatus\n   74.0s        0           90.1          154.9   8053.1  31138.5  34359.7  36507.2 payment\n   74.0s        0           11.0           16.8   5905.6   8589.9  13421.8  13421.8 stockLevel\n   75.0s        0           32.9           14.9  16106.1  40802.2  77309.4  77309.4 delivery\n   75.0s        0           92.7          140.4  16106.1  36507.2  38654.7  38654.7 newOrder\n   75.0s        0           10.0           16.3   2415.9   7516.2   7516.2   7516.2 orderStatus\n   75.0s        0          204.2          155.6   6979.3  30064.8  38654.7  49392.1 payment\n   75.0s        0           12.0           16.7   2080.4   9663.7  12884.9  12884.9 stockLevel\n   76.0s        0           37.1           15.2  14495.5  36507.2  64424.5  64424.5 delivery\n   76.0s        0           22.1          138.9  15032.4  30064.8  32212.3  32212.3 newOrder\n   76.0s        0            8.0           16.2    939.5   4831.8   4831.8   4831.8 orderStatus\n   76.0s        0          244.7          156.7   6710.9  36507.2  38654.7  47244.6 payment\n   76.0s        0           11.0           16.7    805.3   2818.6   6442.5   6442.5 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   77.0s        0           21.0           15.3  12348.0  40802.2  40802.2  40802.2 delivery\n   77.0s        0           12.0          137.2   9663.7  34359.7  38654.7  38654.7 newOrder\n   77.0s        0           12.0           16.2   1879.0  11274.3  12348.0  12348.0 orderStatus\n   77.0s        0          120.0          156.3   1744.8  36507.2  40802.2  40802.2 payment\n   77.0s        0           13.0           16.6   1342.2   4563.4   7247.8   7247.8 stockLevel\n   78.0s        0           14.0           15.2   9663.7  26843.5  57982.1  57982.1 delivery\n   78.0s        0            9.0          135.6   4831.8  16106.1  16106.1  16106.1 newOrder\n   78.0s        0            3.0           16.0    838.9   2818.6   2818.6   2818.6 orderStatus\n   78.0s        0           59.1          155.0   1677.7   2952.8   3087.0   6174.0 payment\n   78.0s        0            7.0           16.5   1208.0   2415.9   2415.9   2415.9 stockLevel\n   79.0s        0           15.0           15.2  12348.0  36507.2  38654.7  38654.7 delivery\n   79.0s        0            4.0          133.9   8589.9  33286.0  33286.0  33286.0 newOrder\n   79.0s        0            1.0           15.8    872.4    872.4    872.4    872.4 orderStatus\n   79.0s        0           30.0          153.4   1744.8   5100.3   9126.8   9126.8 payment\n   79.0s        0            3.0           16.3   3221.2   9126.8   9126.8   9126.8 stockLevel\n   80.0s        0            7.9           15.1  12884.9  45097.2  45097.2  45097.2 delivery\n   80.0s        0            0.0          132.2      0.0      0.0      0.0      0.0 newOrder\n   80.0s        0            0.0           15.6      0.0      0.0      0.0      0.0 orderStatus\n   80.0s        0           26.7          151.9   2147.5   5905.6   7516.2   7516.2 payment\n   80.0s        0            1.0           16.1   3489.7   3489.7   3489.7   3489.7 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   81.0s        0           10.1           15.1  10200.5  45097.2  45097.2  45097.2 delivery\n   81.0s        0           10.1          130.7  27917.3  42949.7  42949.7  42949.7 newOrder\n   81.0s        0            4.0           15.5    939.5  10200.5  10200.5  10200.5 orderStatus\n   81.0s        0           23.2          150.3   6442.5  40802.2  42949.7  42949.7 payment\n   81.0s        0            0.0           15.9      0.0      0.0      0.0      0.0 stockLevel\n   82.0s        0           16.0           15.1  23622.3  38654.7  40802.2  40802.2 delivery\n   82.0s        0          120.8          130.6  26843.5  45097.2  47244.6  49392.1 newOrder\n   82.0s        0           21.0           15.5   9126.8  16106.1  18253.6  18253.6 orderStatus\n   82.0s        0           85.9          149.5  15569.3  24696.1  38654.7  40802.2 payment\n   82.0s        0            9.0           15.8   7784.6  19327.4  19327.4  19327.4 stockLevel\n   83.0s        0           14.0           15.1  20401.1  47244.6  49392.1  49392.1 delivery\n   83.0s        0          219.8          131.7  22548.6  45097.2  47244.6  55834.6 newOrder\n   83.0s        0           25.0           15.7   9663.7  18253.6  19327.4  19327.4 orderStatus\n   83.0s        0           63.9          148.5  14495.5  32212.3  42949.7  42949.7 payment\n   83.0s        0           11.0           15.8   7247.8  19327.4  22548.6  22548.6 stockLevel\n   84.0s        0            8.0           15.0  19327.4  42949.7  42949.7  42949.7 delivery\n   84.0s        0          204.1          132.6  20401.1  47244.6  49392.1  60129.5 newOrder\n   84.0s        0           20.0           15.7   7516.2  16106.1  16643.0  16643.0 orderStatus\n   84.0s        0           47.0          147.2   6174.0  40802.2  45097.2  45097.2 payment\n   84.0s        0           11.0           15.7   2684.4  15032.4  15569.3  15569.3 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   85.0s        0            3.0           14.9   9126.8  16106.1  16106.1  16106.1 delivery\n   85.0s        0          387.3          135.6  22548.6  45097.2  49392.1  60129.5 newOrder\n   85.0s        0           15.0           15.7   1677.7  16643.0  18253.6  18253.6 orderStatus\n   85.0s        0           38.9          146.0   3087.0  12348.0  16106.1  16106.1 payment\n   85.0s        0           11.0           15.7   6442.5  10200.5  21474.8  21474.8 stockLevel\n   86.0s        0           11.0           14.8  21474.8  31138.5  47244.6  47244.6 delivery\n   86.0s        0          319.8          137.7  22548.6  47244.6  49392.1  53687.1 newOrder\n   86.0s        0           15.0           15.7   1342.2   3489.7  15569.3  15569.3 orderStatus\n   86.0s        0           60.0          145.0   4160.7  23622.3  24696.1  25769.8 payment\n   86.0s        0           16.0           15.7   2684.4  10200.5  12884.9  12884.9 stockLevel\n   87.0s        0           10.0           14.8  21474.8  49392.1  49392.1  49392.1 delivery\n   87.0s        0          240.4          138.9  22548.6  49392.1  51539.6  53687.1 newOrder\n   87.0s        0           17.0           15.7   1610.6   5368.7  20401.1  20401.1 orderStatus\n   87.0s        0          180.3          145.4  21474.8  26843.5  40802.2  45097.2 payment\n   87.0s        0           29.0           15.8   1946.2   4160.7  11274.3  11274.3 stockLevel\n   88.0s        0           13.0           14.7  21474.8  45097.2  49392.1  49392.1 delivery\n   88.0s        0          185.0          139.4  23622.3  45097.2  49392.1  51539.6 newOrder\n   88.0s        0            9.0           15.6   2281.7  19327.4  19327.4  19327.4 orderStatus\n   88.0s        0          108.0          145.0  23622.3  27917.3  30064.8  32212.3 payment\n   88.0s        0           19.0           15.9   2281.7   6442.5  12348.0  12348.0 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   89.0s        0           13.0           14.7  25769.8  40802.2  47244.6  47244.6 delivery\n   89.0s        0          254.9          140.7  22548.6  47244.6  49392.1  53687.1 newOrder\n   89.0s        0           18.0           15.7   1677.7  12884.9  16643.0  16643.0 orderStatus\n   89.0s        0          145.9          145.0  23622.3  36507.2  45097.2  47244.6 payment\n   89.0s        0           12.0           15.8   2415.9   2818.6   6174.0   6174.0 stockLevel\n   90.0s        0            7.0           14.6  27917.3  49392.1  49392.1  49392.1 delivery\n   90.0s        0          200.9          141.4  22548.6  47244.6  49392.1  55834.6 newOrder\n   90.0s        0           10.0           15.6   1744.8   7247.8   7247.8   7247.8 orderStatus\n   90.0s        0          217.9          145.8   7516.2  30064.8  36507.2  45097.2 payment\n   90.0s        0           23.0           15.9   2550.1   6174.0   6442.5   6442.5 stockLevel\n   91.0s        0           27.0           14.8  27917.3  51539.6  81604.4  81604.4 delivery\n   91.0s        0          255.1          142.6  20401.1  49392.1  53687.1  53687.1 newOrder\n   91.0s        0           18.0           15.6   1610.6   2415.9   2952.8   2952.8 orderStatus\n   91.0s        0          164.1          146.0   8053.1  38654.7  49392.1  51539.6 payment\n   91.0s        0            9.0           15.8   1811.9   6710.9   6710.9   6710.9 stockLevel\n   92.0s        0            7.0           14.7  27917.3  34359.7  34359.7  34359.7 delivery\n   92.0s        0          318.9          144.5  22548.6  51539.6  55834.6  55834.6 newOrder\n   92.0s        0           18.0           15.6   1476.4   2684.4   7516.2   7516.2 orderStatus\n   92.0s        0           83.0          145.3   5637.1  19327.4  24696.1  27917.3 payment\n   92.0s        0           18.0           15.8   1879.0   4831.8   5637.1   5637.1 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   93.0s        0           24.0           14.8  28991.0  53687.1  55834.6  55834.6 delivery\n   93.0s        0          188.7          145.0  18253.6  51539.6  55834.6  55834.6 newOrder\n   93.0s        0           14.0           15.6   1476.4   5637.1   6174.0   6174.0 orderStatus\n   93.0s        0          377.4          147.8  17179.9  31138.5  51539.6  53687.1 payment\n   93.0s        0           17.0           15.9   2550.1   4026.5   5368.7   5368.7 stockLevel\n   94.0s        0           15.0           14.8  28991.0  51539.6  62277.0  62277.0 delivery\n   94.0s        0          184.1          145.4  18253.6  51539.6  53687.1  55834.6 newOrder\n   94.0s        0           19.0           15.7   1543.5   5368.7   6979.3   6979.3 orderStatus\n   94.0s        0          260.2          149.0   9663.7  28991.0  47244.6  57982.1 payment\n   94.0s        0           26.0           16.0   2684.4   9126.8  10200.5  10200.5 stockLevel\n   95.0s        0           14.0           14.8  19327.4  30064.8  34359.7  34359.7 delivery\n   95.0s        0          137.0          145.3  10737.4  49392.1  57982.1  57982.1 newOrder\n   95.0s        0           27.0           15.8   1476.4   8053.1   9663.7   9663.7 orderStatus\n   95.0s        0          247.1          150.0  10200.5  49392.1  55834.6  57982.1 payment\n   95.0s        0           10.0           15.9   1543.5   7247.8   7247.8   7247.8 stockLevel\n   96.0s        0           21.0           14.8  13958.6  40802.2  57982.1  57982.1 delivery\n   96.0s        0          121.0          145.1  11274.3  51539.6  55834.6  60129.5 newOrder\n   96.0s        0           23.0           15.9   1543.5   4831.8   4831.8   4831.8 orderStatus\n   96.0s        0          204.1          150.6   7784.6  33286.0  51539.6  55834.6 payment\n   96.0s        0           13.0           15.9   2818.6   6710.9   8589.9   8589.9 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n   97.0s        0           21.0           14.9  12348.0  36507.2  40802.2  40802.2 delivery\n   97.0s        0          285.6          146.5  13421.8  51539.6  60129.5  62277.0 newOrder\n   97.0s        0           13.0           15.8   1275.1   3892.3   5905.6   5905.6 orderStatus\n   97.0s        0          173.8          150.8   6710.9  33286.0  47244.6  60129.5 payment\n   97.0s        0           19.0           15.9   3489.7   6442.5   8321.5   8321.5 stockLevel\n   98.0s        0            8.0           14.8   9663.7  14495.5  14495.5  14495.5 delivery\n   98.0s        0           80.2          145.9   9663.7  53687.1  57982.1  60129.5 newOrder\n   98.0s        0           10.0           15.8   2415.9   6979.3   6979.3   6979.3 orderStatus\n   98.0s        0          254.6          151.9  10200.5  55834.6  57982.1  62277.0 payment\n   98.0s        0           34.1           16.1   2415.9   6979.3   7784.6   7784.6 stockLevel\n   99.0s        0            9.0           14.8   7247.8  13421.8  13421.8  13421.8 delivery\n   99.0s        0           25.0          144.6   6979.3   9663.7   9663.7   9663.7 newOrder\n   99.0s        0            4.0           15.6    704.6   1946.2   1946.2   1946.2 orderStatus\n   99.0s        0          100.2          151.4   5100.3  10200.5  13421.8  13421.8 payment\n   99.0s        0           12.0           16.1   2415.9   7784.6  10200.5  10200.5 stockLevel\n  100.0s        0           25.0           14.9  10200.5  32212.3  38654.7  38654.7 delivery\n  100.0s        0           45.9          143.7   9663.7  15569.3  25769.8  25769.8 newOrder\n  100.0s        0            6.0           15.5   1040.2   2684.4   2684.4   2684.4 orderStatus\n  100.0s        0          262.7          152.5   9126.8  57982.1  62277.0  66572.0 payment\n  100.0s        0            2.0           15.9   4295.0   5368.7   5368.7   5368.7 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n  101.0s        0            7.0           14.8   9126.8  27917.3  27917.3  27917.3 delivery\n  101.0s        0          158.1          143.8  10737.4  32212.3  60129.5  64424.5 newOrder\n  101.0s        0           11.0           15.5   4295.0   5637.1   8321.5   8321.5 orderStatus\n  101.0s        0           61.0          151.6   7784.6  31138.5  33286.0  34359.7 payment\n  101.0s        0           12.0           15.9   2550.1   9126.8  14495.5  14495.5 stockLevel\n  102.0s        0           14.0           14.8   8321.5  24696.1  27917.3  27917.3 delivery\n  102.0s        0           76.9          143.1  11274.3  57982.1  64424.5  73014.4 newOrder\n  102.0s        0            6.0           15.4   1275.1   4563.4   4563.4   4563.4 orderStatus\n  102.0s        0          184.9          151.9   9126.8  36507.2  62277.0  73014.4 payment\n  102.0s        0           12.0           15.8   1040.2   5100.3   5637.1   5637.1 stockLevel\n  103.0s        0           12.0           14.8   9126.8  13421.8  15569.3  15569.3 delivery\n  103.0s        0           82.1          142.6  11274.3  17179.9  19327.4  38654.7 newOrder\n  103.0s        0            9.0           15.3   3355.4  10737.4  10737.4  10737.4 orderStatus\n  103.0s        0          118.1          151.6   7516.2  40802.2  62277.0  64424.5 payment\n  103.0s        0           14.0           15.8   1409.3  10737.4  11274.3  11274.3 stockLevel\n  104.0s        0           14.0           14.8  13421.8  36507.2  60129.5  60129.5 delivery\n  104.0s        0          175.9          142.9  11274.3  22548.6  60129.5  64424.5 newOrder\n  104.0s        0           23.0           15.4   3623.9   7784.6  10200.5  10200.5 orderStatus\n  104.0s        0          231.8          152.3  10737.4  16106.1  62277.0  66572.0 payment\n  104.0s        0           11.0           15.8   4295.0  13421.8  15032.4  15032.4 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n  105.0s        0           19.0           14.8  13421.8  23622.3  42949.7  42949.7 delivery\n  105.0s        0          217.5          143.6  12884.9  60129.5  68719.5  73014.4 newOrder\n  105.0s        0           22.9           15.5   1409.3   8321.5   8589.9   8589.9 orderStatus\n  105.0s        0          237.5          153.2   9126.8  21474.8  62277.0  64424.5 payment\n  105.0s        0           14.0           15.8   3087.0   5368.7   8589.9   8589.9 stockLevel\n  106.0s        0           18.0           14.8  12348.0  20401.1  32212.3  32212.3 delivery\n  106.0s        0          199.9          144.1  11811.2  42949.7  64424.5  68719.5 newOrder\n  106.0s        0           25.0           15.6   1677.7   6979.3   9126.8   9126.8 orderStatus\n  106.0s        0           68.0          152.3   5368.7  14495.5  38654.7  45097.2 payment\n  106.0s        0           20.0           15.8   2550.1   6979.3  11811.2  11811.2 stockLevel\n  107.0s        0            6.0           14.7  12348.0  64424.5  64424.5  64424.5 delivery\n  107.0s        0          222.4          144.9  12348.0  40802.2  45097.2  73014.4 newOrder\n  107.0s        0           20.0           15.6   2415.9  14495.5  15569.3  15569.3 orderStatus\n  107.0s        0          172.3          152.5   6710.9  15569.3  40802.2  47244.6 payment\n  107.0s        0           25.0           15.9   2684.4  10200.5  13958.6  13958.6 stockLevel\n  108.0s        0           12.0           14.7  15569.3  42949.7  45097.2  45097.2 delivery\n  108.0s        0          246.4          145.8  11811.2  45097.2  66572.0  73014.4 newOrder\n  108.0s        0           20.0           15.7   7247.8  14495.5  16106.1  16106.1 orderStatus\n  108.0s        0          175.6          152.7   8321.5  17179.9  40802.2  66572.0 payment\n  108.0s        0           12.0           15.8   2415.9  12348.0  15032.4  15032.4 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n  109.0s        0           15.0           14.7  12884.9  42949.7  64424.5  64424.5 delivery\n  109.0s        0          234.5          146.6   9663.7  33286.0  42949.7  47244.6 newOrder\n  109.0s        0           23.0           15.7   1811.9   3623.9   3892.3   3892.3 orderStatus\n  109.0s        0          319.6          154.3  10200.5  16643.0  40802.2  47244.6 payment\n  109.0s        0           13.0           15.8   2281.7   5368.7   9126.8   9126.8 stockLevel\n  110.0s        0           11.0           14.7  12884.9  20401.1  73014.4  73014.4 delivery\n  110.0s        0          145.0          146.6   6442.5  19327.4  42949.7  45097.2 newOrder\n  110.0s        0           17.0           15.7    906.0   2281.7   3489.7   3489.7 orderStatus\n  110.0s        0          325.0          155.8   6174.0  17179.9  45097.2  57982.1 payment\n  110.0s        0           17.0           15.8   1610.6   6174.0  13421.8  13421.8 stockLevel\n  111.0s        0           14.0           14.7  11274.3  22548.6  34359.7  34359.7 delivery\n  111.0s        0          156.0          146.7   6442.5  16643.0  36507.2  40802.2 newOrder\n  111.0s        0           21.0           15.8   1342.2   5368.7  13421.8  13421.8 orderStatus\n  111.0s        0          169.0          155.9   5100.3  12884.9  15569.3  38654.7 payment\n  111.0s        0           21.0           15.9   1811.9  13421.8  24696.1  24696.1 stockLevel\n  112.0s        0           22.0           14.7  10200.5  27917.3  49392.1  49392.1 delivery\n  112.0s        0          262.6          147.7   7784.6  25769.8  45097.2  49392.1 newOrder\n  112.0s        0           17.0           15.8   1342.2   3892.3  19327.4  19327.4 orderStatus\n  112.0s        0          166.8          156.0   4295.0   7784.6   8589.9   9663.7 payment\n  112.0s        0           11.0           15.8   2550.1  10200.5  20401.1  20401.1 stockLevel\n_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)\n  113.0s        0           23.0           14.8   9126.8  51539.6  68719.5  68719.5 delivery\n  113.0s        0          208.4          148.2   6710.9  23622.3  27917.3  38654.7 newOrder\n  113.0s        0           12.0           15.8    939.5   2013.3   2147.5   2147.5 orderStatus\n  113.0s        0          141.3          155.9   5368.7  15032.4  18253.6  18253.6 payment\n  113.0s        0           15.0           15.8   2550.1   8321.5   8589.9   8589.9 stockLevel\n  114.0s        0           18.0           14.9  10200.5  40802.2  49392.1  49392.1 delivery\n  114.0s        0          103.1          147.9   5905.6   9663.7  12348.0  77309.4 newOrder\n  114.0s        0           22.0           15.8   1140.9  16643.0  18253.6  18253.6 orderStatus\n  114.0s        0          202.2          156.3   4026.5   6710.9   8589.9  10200.5 payment\n  114.0s        0           17.0           15.8   1946.2   4026.5   4026.5   4026.5 stockLevel\n  115.0s        0           23.0           14.9   8589.9  11274.3  11811.2  11811.2 delivery\n  115.0s        0          136.0          147.7   6442.5  12348.0  23622.3  24696.1 newOrder\n  115.0s        0           13.0           15.8   1073.7   1677.7   2147.5   2147.5 orderStatus\n  115.0s        0           92.0          155.8   4563.4   8589.9  14495.5  14495.5 payment\n  115.0s        0           21.0           15.9   1946.2   5368.7  15032.4  15032.4 stockLevel\n  116.0s        0           20.0           15.0  10200.5  19327.4  30064.8  30064.8 delivery\n  116.0s        0          168.8

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1739656&tab=buildLog) on [master@c4cf5585cbda55caadb1782f6705e52a83659538](https://github.com/cockroachdb/cockroach/commits/c4cf5585cbda55caadb1782f6705e52a83659538):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpccbench/nodes=9/cpu=4/multi-region/run_1
	cluster.go:2368,tpcc.go:729,tpcc.go:570,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:729
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - unexpected node event: 9: dead

	cluster.go:1410,context.go:135,cluster.go:1399,test_runner.go:778: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1739656-1581235400-86-n12cpu4-geo --oneshot --ignore-empty-nodes: exit status 1 4: skipped
		2: 3315
		3: 4157
		1: 3749
		8: skipped
		12: skipped
		5: 3666
		6: 3811
		7: 3781
		10: 3730
		9: dead
		11: 3740
		Error:  9: dead

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1739656&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44541 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1749385&tab=buildLog) on [master@e671a4ef97cbc6cf5d22f8f322fd45733d302094](https://github.com/cockroachdb/cockroach/commits/e671a4ef97cbc6cf5d22f8f322fd45733d302094):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:774: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1749385&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #45086 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44958 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062122_v19.2.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062122_v19.2.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44541 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1759645&tab=buildLog) on [master@b797cad6d130714748983bc53d4611ddc6151153](https://github.com/cockroachdb/cockroach/commits/b797cad6d130714748983bc53d4611ddc6151153):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:774: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1759645&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #45086 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44958 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062122_v19.2.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062122_v19.2.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44541 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1793930&tab=buildLog) on [master@752dea867f3aeb142e98c22f8d320ce19041aa8d](https://github.com/cockroachdb/cockroach/commits/752dea867f3aeb142e98c22f8d320ce19041aa8d):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200307-1793930/tpccbench/nodes=9/cpu=4/multi-region/run_1
	cluster.go:2368,tpcc.go:729,tpcc.go:570,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:729
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - unexpected node event: 10: dead

	cluster.go:1410,context.go:135,cluster.go:1399,test_runner.go:778: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1793930-1583568484-85-n12cpu4-geo --oneshot --ignore-empty-nodes: exit status 1 4: skipped
		3: 4433
		1: 4573
		2: 4529
		8: skipped
		12: skipped
		7: 4240
		5: 4647
		6: 4230
		10: dead
		9: 4091
		11: 4061
		Error:  10: dead

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1793930&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #45086 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44958 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062122_v19.2.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062122_v19.2.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44541 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1795062&tab=buildLog) on [master@dfa5bd527ae7d7373dd03c62118df87a87a77130](https://github.com/cockroachdb/cockroach/commits/dfa5bd527ae7d7373dd03c62118df87a87a77130):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200308-1795062/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:774: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1795062&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #45086 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44958 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062122_v19.2.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062122_v19.2.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44541 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1796240&tab=buildLog) on [master@c473f40078994551cebcbe00fdbf1fa388957658](https://github.com/cockroachdb/cockroach/commits/c473f40078994551cebcbe00fdbf1fa388957658):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200309-1796240/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:774: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1796240&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #45086 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44958 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062122_v19.2.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062122_v19.2.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44541 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1799071&tab=buildLog) on [master@72c4a1bd411f2f82bf9aaa22883821a946614148](https://github.com/cockroachdb/cockroach/commits/72c4a1bd411f2f82bf9aaa22883821a946614148):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200310-1799071/tpccbench/nodes=9/cpu=4/multi-region/run_1
	cluster.go:2368,tpcc.go:729,tpcc.go:570,test_runner.go:747: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:729
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:747
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:797: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1799071&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #45086 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44958 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062122_v19.2.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062122_v19.2.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44541 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001281357_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001281357_v19.2.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1801614&tab=buildLog) on [master@5570c01402796edb7cd06eb8ce7f615371f22d42](https://github.com/cockroachdb/cockroach/commits/5570c01402796edb7cd06eb8ce7f615371f22d42):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200311-1801614/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:747: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:747
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:797: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1801614&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1808695&tab=buildLog) on [master@33d71472dc01cbc5064b3c5e1fcd666a33f606de](https://github.com/cockroachdb/cockroach/commits/33d71472dc01cbc5064b3c5e1fcd666a33f606de):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200315-1808695/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:803: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1808695&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1811809&tab=buildLog) on [master@5a3d0c9539a671f0e55b680d3021b18dde9d190d](https://github.com/cockroachdb/cockroach/commits/5a3d0c9539a671f0e55b680d3021b18dde9d190d):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200317-1811809/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:803: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1811809&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #46184 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003161814_v19.2.5](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003161814_v19.2.5) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1814553&tab=buildLog) on [master@b5f030223fbcf22e806c48a3c46e74a73a54f50f](https://github.com/cockroachdb/cockroach/commits/b5f030223fbcf22e806c48a3c46e74a73a54f50f):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200318-1814553/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:821: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1814553&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #46184 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003161814_v19.2.5](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003161814_v19.2.5) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44667 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44302 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1823985&tab=buildLog) on [master@055561809b95488bff2cad19422e7f4a7472e3a2](https://github.com/cockroachdb/cockroach/commits/055561809b95488bff2cad19422e7f4a7472e3a2):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200324-1823985/tpccbench/nodes=9/cpu=4/multi-region/run_1
	tpcc.go:858,tpcc.go:570,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.runTPCCBench.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:835
		    github.com/cockroachdb/cockroach/pkg/util/search.searchWithSearcher
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:43
		    github.com/cockroachdb/cockroach/pkg/util/search.(*lineSearcher).Search
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/util/search/search.go:173
		    main.runTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:746
		    main.registerTPCCBenchSpec.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:570
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func3
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2416
		    main.(*monitor).wait.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2445
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor command failure:
		  - signal: interrupt

	cluster.go:2050,cluster.go:2069,cluster.go:2173,cluster.go:1470,context.go:135,cluster.go:1467,test_runner.go:821: context canceled

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1823985&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #46387 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46343 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46338 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003181957_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003181957_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/tpcc.go
+++ b/pkg/cmd/roachtest/tpcc.go
@@ -365,7 +365,7 @@ func registerTPCC(r *testRegistry) {
 		LoadConfig:   multiLoadgen,
 
 		LoadWarehouses: 5000,
-		EstimatedMax:   2200,
+		EstimatedMax:   3000,
 
 		MinVersion: "v19.1.0",
 	})

